travis_install_selenium

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
This script needs a shebang (`!/bin/sh`) and must be executable (`chmod +x travis_install_selenium` Though I don't know if it already is, since I'm not sure how GitHub will report file modes...)